### DISCUSSION TITLE
新增了 睡眠监测 设置选项，关闭后，电脑在跑自动化测试，Tai也能统计时长吗？

### DISCUSSION THREAD

**Discussion by WinsonMok** [Planshit/Tai#304] [Q&A]
新增了 睡眠监测 设置选项，关闭后，电脑在跑自动化测试，Tai也能统计时长吗？即使在晚上，谢谢

**Comment by noberumotto** [comment]
是的 理论上是可以的

**Comment by noberumotto** [comment]
怎么样 请问关闭后能统计到吗

**Reply by WinsonMok** [reply]
可以，但好像是只要那个自动化软件挂在那，即使不跑起来，也会统计，导致统计的时间会很长，可以根据有没有数据交互优化一下吗？谢谢
